src/cow-react/modules/swap/containers/EthFlowStepper/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  EthFlowStepper as Pure,
+  EthFlowStepperProps,
+  SmartOrderStatus,
+} from '@cow/modules/swap/pure/EthFlow/EthFlowStepper'
+import { useDetectNativeToken } from '@cow/modules/swap/hooks/useDetectNativeToken'
+import { Order } from 'state/orders/actions'
+import { NATIVE_CURRENCY_BUY_ADDRESS } from 'constants/index'
+
+type Props = {
+  storeOrder: Order | undefined
```

Comment:
why "store"?

Reply:
Was thinking about the updater where we have store and api orders.
Renamed to order